torch/csrc/jit/mobile/model_compatibility.cpp

```diff
@@ -58,38 +58,38 @@ std::vector<IValue> get_bytecode_ivalues(PyTorchStreamReader& reader) {
 /********************** Bytecode **********************/
 
 // Forward declare
-int64_t _get_model_bytecode_version(
+uint64_t _get_model_bytecode_version(
     const std::vector<IValue>& bytecode_ivalues);
 
-int64_t _get_model_bytecode_version(std::istream& in) {
+uint64_t _get_model_bytecode_version(std::istream& in) {
   std::unique_ptr<IStreamAdapter> rai = std::make_unique<IStreamAdapter>(&in);
   return _get_model_bytecode_version(std::move(rai));
 }
 
-int64_t _get_model_bytecode_version(const std::string& filename) {
+uint64_t _get_model_bytecode_version(const std::string& filename) {
   std::unique_ptr<FileAdapter> rai = std::make_unique<FileAdapter>(filename);
   return _get_model_bytecode_version(std::move(rai));
 }
 
-int64_t _get_model_bytecode_version(std::shared_ptr<ReadAdapterInterface> rai) {
+uint64_t _get_model_bytecode_version(
+    std::shared_ptr<ReadAdapterInterface> rai) {
   if (!check_zip_file(rai)) {
-    TORCH_WARN(
-        "The input model might not be generated from _save_for_mobile()");
-    return -1;
+    TORCH_CHECK(
```

Comment:
cc @xcheng16

Reply:
The feedback I received from FL team is to exit gracefully, instead of throwing error. There will be some validation process running in server and client, and gracefully shutdown seems reasonable to me. That's the reason I've been using `TORCH_WARN`. cc @fbhuba